### PR TITLE
INTERFACES-343: Refactor of Process store - Mobx -> Redux

### PR DESCRIPTION
**Pull request by Chris-ssvlabs**

### CODE REVIEW

File: src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/CancelUpdateFee/CancelUpdateFee.tsx [hunk 1]
```diff
@@ -1,36 +1,31 @@
...
 
 const CancelUpdateFee = () => {
-  const stores = useStores();
   const classes = useStyles();
   const navigate = useNavigate();
-  const processStore: ProcessStore = stores.Process;
-  const process: SingleOperator = processStore.getProcess;
-  const operator: IOperator = process.item;
   const [futureFee, setFutureFee] = useState(0);
+  const process: SingleCluster | undefined = useAppSelector(getProcess);
```
Comment (axelrod-blox) [inline src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/CancelUpdateFee/CancelUpdateFee.tsx:27]: the type of process should be SingleOperator

File: src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/DeclareFee.tsx [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
 import { IncreaseFlowProps } from '~app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/IncreaseFlow';
 import ReactStepper from '~app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/Stepper';
 import { StepperSteps, useStyles } from '~app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/index.styles';
 import BorderScreen from '~app/components/common/BorderScreen';
 import ChangeFeeDisplayValues from '~app/components/common/FeeUpdateTo/ChangeFeeDisplayValues';
 import { ButtonSize } from '~app/enums/Button.enum';
 import { useAppDispatch, useAppSelector } from '~app/hooks/redux.hook';
-import { useStores } from '~app/hooks/useStores';
-import { SingleOperator } from '~app/model/processes.model';
+import { SingleCluster } from '~app/model/processes.model';
```
Comment (axelrod-blox) [inline src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/DeclareFee.tsx:12]: SingleOperator

File: src/app/components/applications/SSV/MyAccount/components/RemoveOperator/RemoveOperator.tsx [hunk 3]
```diff
@@ -1,37 +1,33 @@
...
 import { removeOperator } from '~root/services/operatorContract.service';
+import { getProcess } from '~app/redux/process.slice.ts';
 
 const RemoveOperator = () => {
-  const stores = useStores();
   const navigate = useNavigate();
   const [isChecked, setIsChecked] = useState(false);
-  const processStore: ProcessStore = stores.Process;
-  const process: RegisterOperator = processStore.process;
+  const process: SingleCluster | undefined = useAppSelector(getProcess);
```
Comment (axelrod-blox) [inline src/app/components/applications/SSV/MyAccount/components/RemoveOperator/RemoveOperator.tsx:22]: SingleOperator

File: src/app/components/common/NewWhiteWrapper/components/ValidatorsFlow/ValidatorsFlow.tsx [hunk 4]
```diff
@@ -1,14 +1,12 @@
...
 import { useNavigate } from 'react-router-dom';
-import ProcessStore from '~app/common/stores/applications/SsvWeb/Process.store';
-import { useAppSelector } from '~app/hooks/redux.hook';
-import { useStores } from '~app/hooks/useStores';
+import { useAppDispatch, useAppSelector } from '~app/hooks/redux.hook';
 import { SingleCluster } from '~app/model/processes.model';
 import { getAccountAddress } from '~app/redux/wallet.slice';
 import { longStringShorten } from '~lib/utils/strings';
 import { getClusterHash } from '~root/services/cluster.service';
 import { useStyles } from '../../NewWhiteWrapper.styles';
```
Comment (axelrod-blox) [inline src/app/components/common/NewWhiteWrapper/components/ValidatorsFlow/ValidatorsFlow.tsx:8]: change to absolute path pls

**Review by axelrod-blox — APPROVED** [review]

**Comment by axelrod-blox** [comment]
approved with comments

File: src/app/components/common/ToolTip/ToolTip.tsx [hunk 5]
```diff
@@ -3,7 +3,7 @@ import { useStyles } from './ToolTip.styles';
 import AnchorTooltip from './components/AnchorTooltip';
 
 type ToolTipProps = {
-  text?: string;
+  text?: JSX.Element | string;
```
[the remark this answers is not in the document]
Comment (Chris-ssvlabs) [reply src/app/components/common/ToolTip/ToolTip.tsx:6]: It's already used in the project in 9 other places, but sure, I will look and refactor in those too.
image

File: src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/CancelUpdateFee/CancelUpdateFee.tsx (hunk 1 above)
Comment (Chris-ssvlabs) [reply src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/CancelUpdateFee/CancelUpdateFee.tsx]: Too much copy/pasting... fixed.

File: src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx [hunk 6]
```diff
@@ -96,33 +84,37 @@ const ClusterDashboard = () => {
...
+      <ClusterWarnings cluster={cluster} />
     );
   });
 
   const openSingleCluster = (listIndex: number) => {
-    processStore.setProcess({
-      processName: 'single_cluster',
-      item: sortedClusters[listIndex],
-    }, 2);
+    dispatch(
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx:109]: this looks redundant cause in next line there is setting selected cluster in redux

File: src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/CancelUpdateFee/CancelUpdateFee.tsx [hunk 7]
```diff
@@ -1,36 +1,31 @@
...
 
 const CancelUpdateFee = () => {
-  const stores = useStores();
   const classes = useStyles();
   const navigate = useNavigate();
-  const processStore: ProcessStore = stores.Process;
-  const process: SingleOperator = processStore.getProcess;
-  const operator: IOperator = process.item;
   const [futureFee, setFutureFee] = useState(0);
+  const process: SingleOperator | undefined = useAppSelector(getProcess);
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/CancelUpdateFee/CancelUpdateFee.tsx:27]: use getSelectedOperator from account.slice.ts

File: src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/DeclareFee.tsx [hunk 8]
```diff
@@ -1,31 +1,27 @@
...
 
 const DeclareFee = ({ newFee, oldFee, currentCurrency, getCurrentState }: IncreaseFlowProps) => {
-  const stores = useStores();
   const classes = useStyles({});
   const [isLoading, setIsLoading] = useState(false);
   const dispatch = useAppDispatch();
-  const processStore: ProcessStore = stores.Process;
-  const process: SingleOperator = processStore.getProcess;
-  const operator = process.item;
+  const process: SingleOperator | undefined = useAppSelector(getProcess);
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/DeclareFee.tsx:23]: same comment as above

File: src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/DecreaseFlow.tsx [hunk 9]
```diff
@@ -20,17 +16,16 @@ import LinkText from '~app/components/common/LinkText';
...
-  const stores = useStores();
   const navigate = useNavigate();
   const classes = useStyles({});
-  const processStore: ProcessStore = stores.Process;
   const [buttonText, setButtonText] = useState('Update Fee');
   const [updated, setUpdated] = useState(false);
   const isContractWallet = useAppSelector(getIsContractWallet);
-  const process: SingleOperator = processStore.getProcess;
-  const operator = process.item;
+  const process: SingleOperator | undefined = useAppSelector(getProcess);
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/DecreaseFlow.tsx:27]: same comment as above

File: src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/DecreaseFlow.tsx [hunk 10]
```diff
@@ -44,10 +39,15 @@ const DecreaseFlow = ({ oldFee, newFee, currency }: UpdateFeeProps) => {
       if (res) {
         const newOperatorData = await getOperator(processOperatorId);
         const balance = await getOperatorBalance(newOperatorData.id);
-        processStore.setProcess({
-          processName: 'single_operator',
-          item: { ...newOperatorData, balance },
-        }, 1);
+        dispatch(
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/UpdateFee/components/DecreaseFlow.tsx:42]: use getSelectedOperator from account.slice.ts

File: src/app/components/common/ToolTip/ToolTip.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (Chris-ssvlabs) [reply src/app/components/common/ToolTip/ToolTip.tsx:6]: I replaced the `JSX.Element` with `ReactElement`

**Comment by Chris-ssvlabs** [comment]
Decided to abandon this PR due to big amount of conflicts - review should be continued in https://github.com/ssvlabs/ssv-web/pull/903

**Pull request closed without merging**